app/uk/gov/hmrc/securemessage/repository/SecureMessageRepository.scala

```diff
@@ -69,7 +73,7 @@ class SecureMessageRepository @Inject()(
             .background(true)),
         IndexModel(ascending("status"), IndexOptions().name("status").unique(false))
       ),
-      replaceIndexes = false,
+      replaceIndexes = true,
```

Comment:
Are you sure you need this, could be time consuming in production.
Might want to talk to some of the platform people before applying.

Reply:
@ardwalker yes we need it in prod as well. Yes, it takes time. If you know that you did something like this before and someone stopped you doing, please let me know. thanks